js/search/search.js

```diff
@@ -171,7 +171,7 @@ class Search {
     realTimeIndexing(message) {
         if (!message) {
             log.send(logLevels.ERROR, 'RealTime Indexing: Messages not provided');
-            return new Error('RealTime Indexing: Messages is required');
+            throw new Error('RealTime Indexing: Messages is required');
```

Comment:
I am curious here. Why do we make this check here? Why not in the calling method instead? Also, typo. Should be ```Messages are required```.

Reply:
Just a safety check.

Reply:
Yes, I understand. But why here instead of in the calling method? We could avoid a method call if it fails this check there.